api/routers/auth.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import APIRouter, status
+from starlette.exceptions import HTTPException
+
+from api.models.auth import AuthDTO, Token
+from libs.utils.auth import create_access_token, create_api_user, validate_user_credentials
+
+auth_router = APIRouter()
+
+
+@auth_router.post("/create_api_user", status_code=status.HTTP_204_NO_CONTENT)
+async def create_user(auth_info: AuthDTO):
+    """
+    Creates the API user (if it's not already created) with the given password.
+    """
+    try:
+        create_api_user(auth_info.user, auth_info.password)
+    except Exception as e:
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=str(e))
```

Comment:
Shouldn't we raise this inside create_api_user? or is not ideal because another script calls this outside of the API?

Reply:
The `create_api_user` script can be called directly for the user. In that case doesn't make sense return a HTTP status